tidb-cloud/migrate-from-mysql-using-data-migration.md

```diff
@@ -13,7 +13,14 @@ This feature helps you migrate your database and its ongoing changes to TiDB Clo
 
 - The Data Migration feature is available only for **Dedicated Tier** clusters.
 
-- The Data Migration feature is only available to clusters in the projects that are created in the AWS Oregon (us-west-2), AWS Singapore (ap-southeast-1), and AWS Tokyo (ap-northeast-1) regions after November 9, 2022. If your **project** was created before the date or if your cluster is in another region, this feature is not available to your cluster and the **Data Migration** tab will not be displayed on the cluster overview page in the TiDB Cloud console.
+- The Data Migration feature is only available to clusters in the projects that are created in the following regions after November 9, 2022. If your **project** was created before the date or if your cluster is in another region, this feature is not available to your cluster and the **Data Migration** tab will not be displayed on the cluster overview page in the TiDB Cloud console.
+
+    - AWS Oregon (us-west-2)
+    - AWS N. Virginia (us-east-1)
+    - AWS Mumbai (ap-south-1)
+    - AWS Singapore (ap-southeast-1)
+    - AWS Tokyo (ap-northeast-1)
+    - AWS Frankfurt (eu-central-1)
 
 - Currently, the Data Migration feature is in beta, and you can create only one migration job **for free** for each organization. To create more migration jobs, you need to [file a support ticket](/tidb-cloud/tidb-cloud-support.md).
```

Comment:
```suggestion
```

Reply:
maybe keep this part `To create more migration jobs, you need to [file a support ticket](/tidb-cloud/tidb-cloud-support.md).`

Reply:
OK, updated.